src/Microsoft.DocAsCode.Common/Git/GitUtility.cs

```diff
@@ -60,7 +60,15 @@ public static GitRepoInfo GetRepoInfo(string directory)
         #region Private Methods
         private static GitDetail GetFileDetailCore(string filePath)
```

Comment:
>filePath [](start = 58, length = 8)

is it possible that filePath is a relative path and GetDirectoryName returns empty?